src/direct-binary-upload-options.js

```diff
@@ -167,6 +168,10 @@ export default class DirectBinaryUploadOptions {
     }
 
     /**
+     * DEPRECATED: This method has been deprecated and should no longer be used. The library
```

Comment:
You may want to use the depracate util from node to emit a `DeprecationWarning` (https://nodejs.org/api/util.html#util_util_deprecate_fn_msg_code). There are npm packages that add browser support, too.

Reply:
Thanks for the tip, I've wrapped the method.